Source/Scene/ConeEmitter.js

```diff
@@ -0,0 +1,51 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/Cartesian3',
+        '../Core/Math',
+        './Particle'
+    ], function(
+        defaultValue,
+        Cartesian3,
+        CesiumMath,
+        Particle) {
+    "use strict";
+
+    var ConeEmitter = function(options) {
```

Comment:
Added ParticleEmitter interface class in 76d6ccf266ffeb984f9621d113eba55457017ae1